.scripts/util.sh

```diff
@@ -59,25 +63,66 @@ is_generated() {
     fi
 }
 
+select_browser_caller() {
+    if [[ "$1" = "chromium" || "$BROWSER" = "chromium" ]]; then
+        BROWSER="chromium"
+        SYS_OPTS="\
+        -DBROWSER_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_NUGET_V/build/native/include\
+        -DBROWSER_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_NUGET_V/build/native/x64\
+        -DBROWSER=$BROWSER"
+    elif [[ "$1" = "mswebview2" || "$BROWSER" = "mswebview2" ]]; then
```

Comment:
This just needs be:
```bash
elif [ "$1" = "mswebview2" ]; then
```

`mswebview2` is not the default option, so there is no need to check the `$BROWSER` variable here.